consumers/express.js

```diff
@@ -46,7 +46,9 @@ module.exports = (config) => async (req, res, next) => {
   }
 
   var nonce = session && session.nonce
-  var app = config[session.provider].key
+  // Make sure all the key variations are checked for both OAuth 1/2
+  // https://github.com/simov/grant/blob/master/config/reserved.json
+  var app = config[session.provider].key || config[session.provider].consumer_key || config[session.provider].client_id
```

Comment:
Good catch! Though I don't think OAuth 1.0a is related to OpenID Connect in any way.

Reply:
Yes, you are right,  `OAuth 1.0` is not related, however people can throw `consumer_key/key` into `config` with `oauth: 2`, as I naively did...